warwick-legend.cc

```diff
@@ -74,7 +76,7 @@ int main(int argc,char** argv)
 #endif
   
   // -- Set mandatory initialization classes
-  G4VUserDetectorConstruction* detector = new LGDetectorConstruction;
+  WLGDDetectorConstruction* detector = new WLGDDetectorConstruction;
```

Comment:
Can use `auto` here as the type (pointer-to-WLGDDetectorConstruction) is explicit on the RHS.